modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/common/view_kb_article.jsp

```diff
@@ -56,20 +56,41 @@ if (portletTitleBasedNavigation) {
 
 	<%
 	KBDropdownItemsProvider kbDropdownItemsProvider = new KBDropdownItemsProvider(liferayPortletRequest, liferayPortletResponse);
+
+	KBArticleDisplayContext kbArticleDisplayContext = new KBArticleDisplayContext(liferayPortletRequest, liferayPortletResponse);
 	%>
 
 	<div class="management-bar management-bar-light navbar navbar-expand-md">
 		<clay:container-fluid>
 			<ul class="navbar-nav navbar-nav-expand">
-				<li class="m-auto nav-item">
-					<aui:workflow-status markupView="lexicon" showHelpMessage="<%= false %>" showIcon="<%= false %>" showLabel="<%= false %>" status="<%= kbArticle.getStatus() %>" version="<%= String.valueOf(kbArticle.getVersion()) %>" />
+				<li class="nav-item nav-item-expand"></li>
```

Comment:
Looks too much hacky to manage the vertical alignment. I think that we can found a better alternative

Reply:
After researching about [aligning things in flexbox](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Aligning_Items_in_a_Flex_Container) , I added the [`justify-content-end`](https://clayui.com/docs/css/utilities/markup-flex.html#css-justify-content) and looks good

![image](https://user-images.githubusercontent.com/67901/207320231-e587d22e-c7da-4856-a521-e78a40535c0a.png)

Reply:
Thank you. Even if I don't like having empty nodes, I saw that uses in portal, but your suggestion is much better